src/commands/react-native/xcode.ts

```diff
@@ -198,7 +207,7 @@ export class XCodeCommand extends Command {
   }
 
   private bundleReactNativeCodeAndImages = async () => {
-    const bundleJSChildProcess = spawn(this.scriptPath, [], {
+    const bundleJSChildProcess = spawn(this.scriptPath || getDefaultScriptPath(), [], {
```

Comment:
I think this will make typing a bit more complex, as in the `bundleReactNativeCodeAndImages` function `this.scriptPath` could still be `undefined`, so I will need an extra check.
Do you see an easy way around this?

If I ever want to reuse `this.scriptPath` somewhere else where it should be defined, I will still get a type error so the bug risk is pretty low I think, so I think it is acceptable to keep it as it is.
Is it ok for you if we look for a refactoring if we ever need to reuse this variable?

Reply:
I didn't realize that yes, this would still be `undefined`... so yes, that's ok for me 👍